euth_wagtail/assets/scss/components/_howto.scss

```diff
@@ -0,0 +1,110 @@
+.howto__container {
+    background:
+        linear-gradient(
+            122deg,
+            rgba(72, 196, 200, 1) 11%,
+            rgba(33, 109, 157, 1) 65%,
+            rgba(37, 117, 161, 1) 80%,
+            rgba(72, 168, 200, 1) 93%
+        );
+    height: 100vh;
+    display: flex;
+    align-items: flex-start;
+}
+
+.howto-carousel__container {
+    padding-top: 6 * $spacer;
+    width: 100%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.howto-item__container {
+    color: $bright;
+    padding: 0 2 * $spacer;
+
+    & h1 {
+        font-size: 24px;
+        text-align: center;
+        margin-bottom: 3 * $spacer;
+    }
+
+    & h2 {
+        font-size: 18px;
+        text-align: center;
+        margin-top: 4 * $spacer;
+        margin-bottom: $spacer;
+    }
+
+    & p {
+        font-size: 16px;
+        text-align: center;
+    }
+}
+
+img.howto-item__image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 70%;
+    max-width: 300px;
+}
+
+// slick dots overwrites
+ul.slick-dots {
+    margin: 0;
```

Comment:
any slick overwrites should be nested under custom class name of the specific slick carousel to ensure no styles leak to other slicks on the site

Reply:
true 🤝